config/www/cgi/newcgi.py

```
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+def main():
```

Ruff

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)